ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -317,6 +317,8 @@ objects:
           cp "${SSH_PRIVATE_KEY_PATH}" ~/.ssh/id_rsa
           chmod 0600 ~/.ssh/id_rsa
 
+          prepare_ansible_playbooks
```

Comment:
Might be worth adding a comment like "needed for 4.4", so that we know we can delete after 4.7

Reply:
done